pkg/render/manager.go

```diff
@@ -353,6 +354,7 @@ func (c *managerComponent) managerEnvVars() []corev1.EnvVar {
 		{Name: "ENABLE_MULTI_CLUSTER_MANAGEMENT", Value: strconv.FormatBool(c.cfg.ManagementCluster != nil)},
 		// Kibana does not have a FIPS compatible mode, therefore we disable the button in the UI.
 		{Name: "ENABLE_KIBANA", Value: strconv.FormatBool(!operatorv1.IsFIPSModeEnabled(c.cfg.Installation.FIPSMode))},
+		{Name: "ENABLE_COMPLIANCE_REPORTS", Value: strconv.FormatBool(c.cfg.Compliance != nil)},
```

Comment:
Why doesn't this one also depend on the License for Compliance?

Reply:
See my [response](https://github.com/tigera/operator/pull/2111#discussion_r953118710) to your earlier comment, let me know if it answers your question.

As an aside, I left a code comment in the manager test explaining this. If we keep it this way, perhaps it would be worth  moving that here?